modules/nodes-cluster-overcommit-node-memory.adoc

```diff
@@ -13,22 +13,69 @@ from lower OoS classes from using resources requested by pods in higher QoS clas
 By reserving resources for higher QOS levels, pods that don't have resource limits are prevented from encroaching on the resources
 requested by pods at higher QoS levels.
 
-.Procedure
+.Prerequisites 
+	
+. Obtain the label associated with the static Machine Config Pool CRD for the type of node you want to configure. 
+Perform one of the following steps:
+	
+.. View the Machine Config Pool:
++
+[source,bash]
+----
+# oc describe machineconfigpool <name>
+----
++
+For example:
++
+[source,yaml]
+----
+# oc describe machineconfigpool worker
+
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: 2019-02-08T14:52:39Z
+  generation: 1
+  labels:
+    custom-kubelet: small-pods <1>
+----
+<1> If a label has been added it appears under `labels`.
 
-To configure the `qos-reserved` parameter, edit the appropriate node configuration map.
+.. If the label is not present, add a key/value pair:
++
+[source,bash]
+----
+oc label machineconfigpool worker custom-kubelet=small-pods
+---- 
 
+.Procedure
+
+. Create a Custom Resource (CR) for your configuration change.
```

Comment:
We used "CRD" in modules/nodes-nodes-garbage-collection-configuring.adoc
Do we need to make it consistent?

Reply:
@weinliu Yes. They are both Custom Resources. I updated `modules/nodes-nodes-garbage-collection-configuring.adoc`. https://github.com/openshift/openshift-docs/pull/14580